qa/tasks/scripts/ses_rack_dc_region_unavailability.sh

```diff
@@ -0,0 +1,249 @@
+set -ex
+
+master=$(hostname -f)
+
+# split nodes
+split_nodes(){
+
+first_part=$(($1 / 2))
+second_part=$(($1 - $first_part))
+
+}
+
+# wait for cluster health OK
+cluster_health(){
```

Comment:
for better readability, could you please add a 'function' before the function name.